tests/features/solution/add_solution.feature

```diff
@@ -247,3 +247,36 @@ Feature: "Add solution" visibility options.
     Then I delete the "Climate change tracker" solution
     And I delete the "Climate change tracker" solution
     And I delete the "Angela Crespi" contact information
+
+  Scenario: Check solution downloads counter
+    # Create the dummy data to work with.
+    Given the following collection:
+      | title | Ocean studies |
+    Given the following solution:
+      | title       | Climate change tracker                            |
+      | description | Atlantic salmon arrived after the Little Ice Age. |
+      | collection  | Ocean studies                                     |
+      | state       | validated                                         |
+    Given the following distributions:
+      | title                 | description          | parent                 | downloads |
+      | Sample distribution   | Sample description   | Climate change tracker | 10        |
+      | Sample distribution 2 | Sample description 2 | Climate change tracker | 0         |
+    # Checks solutions tiles to we have the correct numbers.
+    Then I visit "/solutions"
+    Then I should see the text "10" in the "Climate change tracker" tile
+    When I go to the homepage of the "Climate change tracker" solution
+    # Check the same thing on the homepage of the solutions.
+    Then I should see the text "Downloads: 10"
+    When I delete the "Sample distribution" asset distribution
+    And I reload the page
+    # Check it again after the first distribution is deleted and cache has been cleared.
+    Then I should not see the text "Downloads: 10"
+    # Go back to the solution tiles page and check downloads are disappeared.
+    Then I visit "/solutions"
+    Then I should not see a "icon--download" element
+    Given the following distribution:
+      | title       | Sample distribution 3  |
+      | description | Sample description 3   |
+      | parent      | Climate change tracker |
+      | downloads   | 20                     |
+    Then I should see the text "20" in the "Test solution" tile
```

Comment:
The first problem here is that, after adding the new distribution, you should reload the page. Otherwise  how can you see the changes?

The 2nd is that you are trying to see the downloads in a tile that doesn't exist: s/Test solution/Climate change tracker.

However, I confirm that the test fails is randomly on the last line. Seems that still a caching issue.

Reply:
Well I think we should resolve the caching thing and then we can adopt the tests as well.

Reply:
This is working fine now and the cache clearing was fixed but now moved to a follow up ticket because it might invalidate the page too often.